docs: add "ivy" and "zone.js" to known scopes in contributing guide

devversion opened the pull request:
Currently the contributing guide misses entries for the `ivy` and `zone.js`
scopes. This commit adds these to the contributing guide as it is useful
for new contributors to know which scopes are supported.

**Note**: master-only because `zone.js` is not ported into the patch branch. The ivy scope could be added to the patch branch.. and therefore it must be part of a patch-only PR.

gkalpak commented on CONTRIBUTING.md:
```diff
@@ -213,6 +213,7 @@ The following is the list of supported scopes:
 * **elements**
 * **forms**
 * **http**
+* **ivy**
```
This is not a package namec (see the description above). It should go in the "exceptions" list below.

devversion replied:
Done. good catch!

gkalpak reviewed (changes requested):
:100:

devversion committed:
docs: add "ivy" and "zone.js" to known scopes in contributing guide

Currently the contributing guide misses entries for the `ivy` and `zone.js`
scopes. This commit adds these to the contributing guide as it is useful
for new contributors to know which scopes are supported.

gkalpak reviewed (approved):
👌

gkalpak commented on CONTRIBUTING.md:
```diff
@@ -231,6 +232,7 @@ There are currently a few exceptions to the "use package name" rule:
 * **changelog**: used for updating the release notes in CHANGELOG.md
 * **docs-infra**: used for docs-app (angular.io) related changes within the /aio directory of the
   repo
+* **ivy**: used for changes to the [Ivy renderer](https://github.com/angular/angular/issues/21706).
```
Nit: Other entries don't have a period at the end 👼

devversion replied:
I explicitly checked but it wasn't consistent. See bullet point below which has a period at the end 😛

gkalpak replied:
👏 👏 👏 Good job! 💯 🥇 
(I'm a little embarrassed now 😛)
